lib/Differentiator/ReverseModeVisitor.cpp

```diff
@@ -1481,9 +1831,17 @@ namespace clad {
   }
 
   StmtDiff ReverseModeVisitor::DifferentiateSingleStmt(const Stmt* S,
-                                                       Expr* dfdS) {
+                                                       Expr* dfdS,
+                                                       bool doNotEmit) {
```

Comment:
```suggestion
                                                       bool shouldEmit/*=true*/) {
```

Don't forget to fix the class declaration.